sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -537,6 +543,14 @@ export class ApplicationStudentsController extends BaseController {
         );
       appealStatus = mostRecentAppeal?.status;
     }
+    let mostRecentApplicationOfferingChangeRequestStatus: ApplicationOfferingChangeRequestStatus;
+    if (application.applicationStatus === ApplicationStatus.Completed) {
+      mostRecentApplicationOfferingChangeRequestStatus =
```

Comment:
@ann-aot @andrewsignori-aot @guru-aot I have a question here. 

Do we need the `mostRecentApplicationOfferingChangeRequestStatus` and `appealStatus` returned as a part of `application/in-progress` API?

I believe they are required only for `application/completed` API? as these values are used only by Completed.vue right?

Let me know if I am missing something

Reply:
I believe, this is not required here. You are right @dheepak-aot

Reply:
you are correct @dheepak-aot. But just wondering if this is `application/${applicationId}/in-progress` endpoint?

Reply:
I am not sure if I am following this, but this method returns information for the "application progress"  not the application "in progress" status. This method can be called at any time (any application status) to provide overall statuses for the application, mainly used for the "application tracker/progress bar", hence we need all the information including appeals and offering request status.
Please let me know If I am missing something.

Reply:
Yes @andrewsignori-aot , you are right.

Reply:
I am sorry guys. I totally lost track of difference between the APIs `in-progress` and `progress-details`. (I created the `progress-details` API I believe 😊) I can recollect now the progress details was to support the progress of the bar.